__tests__/src/index.spec.js

```diff
@@ -24,7 +25,7 @@ describe('toMatchImageSnapshot', () => {
       existsSync: jest.fn(),
       unlinkSync: jest.fn(),
     });
-    mockFs.existsSync.mockImplementation(path => path === 'test/path');
+    mockFs.existsSync.mockImplementation(p => p === 'test/path');
```

Comment:
can you revert this please?

Reply:
No can do, we needed node path in scope.

https://github.com/americanexpress/jest-image-snapshot/pull/23/files/fe94db225f79067516591f142931f73e71b776c6#diff-4fbb28a3592351427bd96f04e7b638a4R16

Reply:
Gotcha! Makes sense.